chainstate/test-suite/src/tests/syncing_tests.rs

```diff
@@ -333,3 +333,24 @@ fn filter_already_existing_blocks_detached_headers(#[case] seed: Seed) {
         );
     });
 }
+
+#[rstest]
+#[trace]
+#[case(Seed::from_entropy())]
+fn initial_block_download(#[case] seed: Seed) {
+    utils::concurrency::model(move || {
+        let mut rng = make_seedable_rng(seed);
+        let mut tf = TestFramework::builder(&mut rng).build();
+
+        // Only genesis block, so is_initial_block_download should return true.
+        assert!(tf.chainstate.is_initial_block_download().unwrap());
+
+        // The added block will have a very fresh timestamp.
```

Comment:
I think that by default the TestFramework uses simulated time. Please use simulated and controlled time with controlled current-time to simulate the cases true/false for initial block download.